tempesta_fw/http.c

```diff
@@ -742,6 +743,53 @@ tfw_http_error_resp_switch(TfwHttpReq *req, int status, const char *reason)
 	tfw_http_send_resp(req, code);
 }
 
+static inline void
+tfw_http_srv_hmonitor(TfwHttpResp *resp)
+{
+	u32 crc32;
+	char *body_cstr;
```

Comment:
Perhaps it's best if these two are be moved to the code block where they are used.

Reply:
This code has been redesigned.